packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -384,7 +384,7 @@ export class GumbyController {
                 this.messenger.sendJobSubmittedMessage(message.tabID)
                 break
             case ButtonActions.STOP_TRANSFORMATION_JOB:
-                await stopTransformByQ(transformByQState.getJobId(), CancelActionPositions.Chat)
```

Comment:
Knowing where a user stopped a job from is not useful and nobody ever looked at this metric